stdlib/tty.pyi

```diff
@@ -17,3 +17,12 @@ if sys.platform != "win32":
     CC: int
     def setraw(fd: _FD, when: int = 2) -> None: ...
     def setcbreak(fd: _FD, when: int = 2) -> None: ...
+
+    if sys.version_info >= (3, 12):
+        __all__ += ["cfmakeraw", "cfmakecbreak"]
```

Comment:
Nit: My quick random sampling shows me that we usually move these `__all__ +=` constructs to the top of the file, even if it means a duplicated version/platform check. Personally, I find this better too, since you can easier spot the contents of `__all__` for all branches.